### ISSUE TITLE
Export to GenAlEx field used for first column 

### ISSUE THREAD

**Issue by doridick14** [genegis/genegis#99]
Export currently puts the Sample_ID in the first column.  But it would be more useful to have the first column be Individual_ID in most cases.

Suggest 2 options that could be used:
1. always put the Individual_ID as the first column but also bring the Sample_ID and  place at the end of the data set.
2. allow user to indicate which field they want to be in the first column but also bring over the other field at the end of the data set

The second option, would be the most useful because it would provide more options to researchers who may not have known individuals.

**Comment by scw** [comment]
Hi Dori,

I've updated the default ordering on the Classified Import step to reflect this change. Are you specifically talking about the Export To SRGD tool, or other export tools? I think generally, we just retain the same ordering of the columns that the user provides us. So if they import an SRGD CSV file with Sample_ID as the first column, we retain that and show the same thing in the results, and don't modify column ordering for example (excepting in formats like GenAlEx where we have to do some manipulation). Happy to change this, but a little more context would be useful.

**Comment by doridick14** [comment]
Hi Shaun,
I was referring specifically to the Export to GenAlEx tools.  I don't think the column order matters for data import into ArcGIS  and I agree that we should retain the order of columns the user provides.  The critical thing for the GenAlEx export is that the column with data in it (either the Individual_ID or Sample_ID) needs to be the 1st column in the export so that it provides some reference for the user.

In the case of SPLASH, the Individual_ID field is more relevant because the data has been extended such that photo records will not have a Sample_ID but might still also have genetic data (linked by Individual_ID) that one might want to include in the Export.

This might not be the case for other users as they might not have known identities and thus only the Sample_ID field will be important for data export to GenAlEx.

Because it really depends on the user's need, I suggested using one of the two options above.

**Comment by scw** [comment]
Hi Dori,

I've added a new option which shows up in the GUI for a 'primary identification field'. It looks for any field ending in '_id', '_ID', or identified as an ID column on import (the columns the user places into the 'Identification Columns' category). It will still default to whatever the primary ID is (usually Individual_ID), but can be changed on export to whatever is most appropriate for analysis. I've built documented and tested this, let me know if it works on your end as well.
